packages/frontend/src/routes/WalletWrapper.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { Wallet } from '../components/wallet/Wallet';
+import { useFungibleTokensIncludingNEAR } from '../hooks/fungibleTokensIncludingNEAR';
+import { Mixpanel } from '../mixpanel/index';
+import { selectAccountId, selectBalance } from '../redux/slices/account';
+import { selectAvailableAccounts } from '../redux/slices/availableAccounts';
+import { selectCreateFromImplicitSuccess, selectCreateCustomName, actions as createFromImplicitActions } from '../redux/slices/createFromImplicit';
+import { selectLinkdropAmount, actions as linkdropActions } from '../redux/slices/linkdrop';
+import { selectTokensWithMetadataForAccountId, actions as nftActions } from '../redux/slices/nft';
+import { actions as tokensActions, selectTokensLoading } from '../redux/slices/tokens';
+
+const { fetchNFTs } = nftActions;
+const { fetchTokens } = tokensActions;
+const { setLinkdropAmount } = linkdropActions;
+const { setCreateFromImplicitSuccess, setCreateCustomName } = createFromImplicitActions;
+
+export function WalletWrapper({
+    tab,
+    setTab
+}) {
+
+    const accountId = useSelector((state) => selectAccountId(state));
+    const balance = useSelector((state) => selectBalance(state));
```

Comment:
:nitpick: since these aren't parameterized selectors
```suggestion
    const accountId = useSelector(selectAccountId);
    const balance = useSelector(selectBalance);
```